src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -1824,4 +1825,41 @@
         </Setter>
     </Style>
 
+    <Style x:Key="LibraryPopupContentStyle"
```

Comment:
Common grey part of popup (with grey triangle):
![image](https://cloud.githubusercontent.com/assets/8158404/10756249/69c69616-7cab-11e5-9a95-cda5708de8e6.png)
![image](https://cloud.githubusercontent.com/assets/8158404/10756253/74bcb9c4-7cab-11e5-9db8-cfd6239fa424.png)